core/input/generisdata_tech.prn

```diff
@@ -96,7 +96,14 @@ inco0               1800         360         360          60          60
 mix0                1.00        1.00        1.00        1.00        1.00        1.00        1.00       
 eta                 0.95        0.85        0.85        1.00        1.00        1.00        1.00   
 omf                 0.03        0.02        0.02        0.03        0.03        0.02        0.06    
-lifetime              45          45          45          30          30          45          45       
+lifetime              45          45          45          30          30          45          45      
```

Comment:
Again some values where it would be interesting to know where they come from... Just a general comment.